static/includes/apps/AppsDatasets.md

```diff
@@ -0,0 +1,63 @@
+&NewLine;
+
+**ix-apps Dataset**
+
+TrueNAS 24.10 and newer creates a hidden **ix-apps** dataset, mounted at <file>/mnt/.ix-apps</file>, to store Docker configuration, catalog data, and app metadata.
+The ix-apps dataset is internally managed by TrueNAS and hidden to prevent user misconfiguration.
```

Comment:
Why are we not using bold for ix-apps dataset in this sentence and the rest of the article? If we are discussing this dataset in general in this content, we can skip the use of bold for ix-apps and ix-applications. We are not pointing to the UI mentions in particular. Thoughs?

Reply:
My thinking was doing so once emphasizes that this is a name that exists in TrueNAS, so it is an element in that sense, but after the initial use we're discussing it generally and it would be a bit much to be bolding it again and again.